mantidproject/mantid
# Fix Flake8 warnings associated with reduction GUI

As part of #20508, 2 files associated with the reduction GUI have been refactored to reduce cyclomatic complexity

**To test:**
Make sure all unit and system tests pass, and code review. In addition, if this is your area, feel free to throw some data at me to test the affected scripts with, to make sure I haven't broken anything

Address part of #20508 . 

**Release Notes** 
*Does not need to be in the release notes as the change is internal*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
